meshtastic/mesh_interface_runtime/node_presentation.py

```
@@ -0,0 +1,211 @@
+    Parameters
+    ----------
+    ts : int | float | None
+        Seconds since the Unix epoch. If `ts` is `None` or `0`, no timestamp is available.
+
+    Returns
+    -------
+    str | None
+        Formatted timestamp string, or `None` if `ts` is `None` or `0`.
+    """
+    return (
+        datetime.fromtimestamp(ts).strftime("%Y-%m-%d %H:%M:%S")
```

Ruff (0.15.7)

[warning] `datetime.datetime.fromtimestamp()` called without a `tz` argument

(DTZ006)